README.md

```diff
@@ -1,2 +1,32 @@
 # auto-video-transcoder
 Automatic video transcoder working on uploaded videos to S3
+
+How to use
+===
+
++ Install ffmpeg ( recommended version 2.4.9-1) e.g yum install ffmpeg
++ Build project 
+
+```
+mvn clean install -DskipTests=true
+```
+
++ add depecndecy to your project 
+
+```
+<dependency>
+			<groupId>com.taakestan</groupId>
+			<artifactId>auto-video-transcoder</artifactId>
+			<version>1.0</version>
```

Comment:
Use snapshot versioning for development, otherwise maven will only download it once and forever! In this case, 1.0-SNAPSHOT will be good.